amundsen_application/static/js/components/common/EditableText/index.tsx

```diff
@@ -136,9 +127,21 @@ class EditableText extends React.Component<EditableTextProps, EditableTextState>
                   </div>
                 </Tooltip>
               </Overlay>
-            </div>
           }
-          <div className='editable-text'>{ this.state.value }</div>
+          <div className={"editable-text"}>
+            { this.state.value }
+            <a className={ "edit-link " + (this.state.value ? "" : "no-value") }
+               href="JavaScript:void(0)"
+               onClick={ this.enterEditMode }
+               ref={ anchor => {
+                  this.editAnchorTarget = anchor;
+                }}
+            >
+              {
+                this.state.value ? "edit" : "Add Description"
```

Comment:
nit: Is the word "edit" also supposed to be capitalized?

Reply:
Personally I would like to use consistent capitalization, but I brought this up with Matt and he specifically asked for this behavior.